src/lib/inputs/InputRadio/InputRadio.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import {
+  Radio as MuiRadio,
+  RadioProps,
+  FormControlLabel as MuiFormControlLabel,
+  FormControlLabelProps
+} from '@material-ui/core';
+import classNames from 'classnames';
+import { FormikProps } from 'formik';
+
+export type InputRadioProps<T> = {
```

Comment:
I'm not sure the advantage of having <T> here when the only time it is used is when it's "any",

this could just be `export type InputRadioProps = { ...`
and then `formikProps?: FormikProps<any>;`

what do you think about that?